python.d/logind.chart.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Description: logind netdata python.d module
+# Author: Austin S. Hemmelgarn (Ferroin)
+# SPDX-License-Identifier: GPL-3.0+
+
+from bases.FrameworkServices.ExecutableService import ExecutableService
+
+PRIORITY = 40000
```

Comment:
`PRIORITY` is wrong

https://github.com/firehol/netdata/blob/master/plugins.d/python.d.plugin#L142-L143
https://github.com/firehol/netdata/blob/master/plugins.d/python.d.plugin#L37-L42

Not sure about 40000 btw too far from 60000

Reply:
Yeah, meant to ask about that.  The point here was to get it just below the `uses` chart, as that seems to me to be the most logical location for it.

Reply:
anyway `priority` should be lowercase

Reply:
40k is ok for now

Reply:
Of course I see this right _after_ I push a commit updating it to 59999 so it just floats to the top of the Python plugins...

I actually am interested to hear @ktsaou's input on what would be reasonable for this.  Personally, I think it actually makes the most sense up with the main system charts (that is, at least above the apps.plugin charts), but I'm not sure where up there, or if it even makes sense to put it all the way up there.